coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -205,3 +205,17 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
                                     marker, range)
+
+    def assemble(self):
+        """
+        Assembles parsed documentation to the original documentation.
+
+        This function assembles the whole documentation comment, with the
+        given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = self.indent + self.marker[0]
+        assembled += ''.join('\n' if line == '\n' and not self.marker[1]
+                             else self.indent + self.marker[1] + line
+                             for line in lines)
+        return assembled + self.indent + self.marker[2]
```

Comment:
what if the last line contains no `\n`? then you overindent it inside the doccomment:
e.g.
```
    /** this is my comment **/
```
would be assembled to
```
    /** this is my comment     **/
```
\+ please test for this case :)